pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -4458,3 +4500,14 @@ func withDomainClock(clock *v1.Clock) libvmi.Option {
 		vmi.Spec.Domain.Clock = clock
 	}
 }
+
+type networkValidatorStub struct {
+	statusCause *metav1.StatusCause
```

Comment:
Please consider having a `[]metav1.StatusCause` here, as it would make `networkValidatorStub.ValidateCreation` even simpler.

Reply:
Done